### ISSUE TITLE
Atomatically hide the taskbar not working correctly on my main display - Taskbar: Windows 10 (ExplorerPatcher)

### ISSUE THREAD

**Issue by WoofGrrrr** [valinet/ExplorerPatcher#3821]
ExplorerPatcher 22621.3880.66.6 (latest)
Windows 11 24H2 OS build 26100.2033
Taskbar style is set to "Windows 10 (ExplorerPatcher)"
"Automatically hide the taskbar" is checked

When I move the cursor down to the bottom of my **main** display, a taskbar is shown
When I move the cursor up, away from the taskbar, the taskbar remains.
Even when I move the cursor to another display
Even when a taskbar is shown on the other display

The taskbar on my main display is hidden ONLY when I move my cursor over an app.

Window focus on cursor is enabled
Window raise on cursor is NOT

Automatically hide the taskbar works correctly on my other displays.

**Comment by pyrates999** [comment]
Try the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases

It doesn't list that it fixes this, but it does fix other things.

**Comment by WoofGrrrr** [comment]
Thanks.  I installed it.  I'm sorry to say that it's still not working correctly

**Comment by Amrsatrio** [comment]
Okay I need more info. What's your display configuration?

**Comment by WoofGrrrr** [comment]
I have three Acer S232HL 23" monitors (yes, they're old.  So am I :-))

The desktop is extended to all three,

Each one is scaled 100%.
Each is set to the recommended resolution: 1920x1080,
Each is set set to Landscape orientation.
They are side-by-side

The IDs are 2, 3, 1.
3 is the Main display.


Display information for all three:
- Desktop mode: 1920 x 1080, 60hz
- Active signal mode: 1920 x 1080, 60hz
- Variable refresh rate: not supported
- Bit depth: 8-bit
- Color format: YCbCr444
- Color space: Standard dynamic range (SDR)
- Refresh Rate: 60Hz
- Dynamic refresh rate isn't supported

Video Card: AMD Radeon RX 580X (Polaris 20 XT) [Pegatron]

Adapter type: Radeon RX 580
- Chip Type: AMD Radeon Graphics Processor (0x67DF)
- DAC Type: Internal DAC(400MHz)
- Adapter String: Radeon RX 580 Series
- BIOS Information: 113-EXT24704-001
- Total Available Graphics Memory: 24442 MB
- Dedicated Video Memory: 8170 MB
- System Video Memory: 0MB
- Shared System Memory: 16271 MB

Taskbar:
- Search: Hide
- Task view: Off
- Widgets: Off
- [x] Automatically hide the taskbar
- [ ] Show badges on taskbar apps
- [ ] Show flashing on taskbar apps
- [x] Show my taskbar on all displays
- When using multiple displays, show my taskbar apps on: Taskbar where window is open
- [ ] share any window from my taskbar
- [ ] Select the far corner of the taskbar to show the desktop
- Combine taskbar buttons and hide labels: Never
- Combine taskbar buttons and hide labels on other taskbars: Never

**Comment by WoofGrrrr** [comment]
> And I can say this: The 24H2 update isn't fully supported

I'm not sure what that means?  That "Taskbar style: Windows 10 (ExplorerPatcher)"  (the custom implementation of the Taskbar) ia still a work in progress???

**Comment by Amrsatrio** [comment]
@Panzimy Yeah you're correct. Some things are beyond repair in 24H2 such as the Battery flyout and the app name for legacy tray notification popups.

@WoofGrrrr No it's not. I daily drive 24H2 on my laptop and the taskbar has been working fine. The taskbar reached completion in July and keeps receiving bug fixes every week since then so why would it still be work in progress??

**Comment by WoofGrrrr** [comment]
> @Panzimy Yeah you're correct. Some things are beyond repair in 24H2 such as the Battery flyout and the app name for legacy tray notification popups.
> 
> @WoofGrrrr No it's not. I daily drive 24H2 on my laptop and the taskbar has been working fine. The taskbar reached completion in July and keeps receiving bug fixes every week since then so why would it still be work in progress??

I'm sorry.  I didn't mean to ruffle any feathers.  I use ep all the time on three PC's and it's working great for me as well.  Except for auto-hide.  And it's not a big deal.  I can just move my cursor over an open window.  I just thought someone might want to know about it.

I was asking what "The 24H2 update isn't fully supported" means.  I must had read into that a "yet".  I thought it meant it *would* be fully supported some time later, as this is the first full release. My mistake.

Now that you have said "Some things are beyond repair in 24H2" I understand better.  It seems that you are saying that it's not possible to fix this problem with Taskbar Auto-Hide???

And this version of ep takes away a problem that I and many other people have been having with the Windows Taskbar: Auto-Hide keeps getting turned off.  This is a HUGE win to me!

Thanks!
